test/check-machines-create

```diff
@@ -1165,6 +1174,71 @@ vnc_password= "{vnc_passwd}"
 
             return self
 
+        def checkMiddlePageWhenCreating(self):
+            b = self.browser
+
+            buttonSelector = "#create-and-run" if self.create_and_run else "#create-and-edit"
+            b.click(buttonSelector)
+
+            if not self.create_and_run:
+                b.wait_visible(f"h1:contains(\"Creating VM {self.name}\")")
+                b.click("button:contains(\"Go to VMs list\")")
+                b.wait_visible("#virtual-machines-listing")
+            b.wait_in_text(f"#vm-{self.name}-system-state", "Creating VM")
+
+            b.click(f"#vm-{self.name}-system-name")
+            b.wait_visible(f"h1:contains(\"Creating VM {self.name}\")")
```

Comment:
I'm wondering what this PR adds, so far I can see that this is a new test assertion, not covered by tests:

```
[jelle@t14s][~/projects/cockpit-machines]%git grep "Creating VM" test/
[jelle@t14s][~/projects/cockpit-machines]%
```

But we can easily move this into an existing test?

Reply:
This test check the middle page after clicking "Create and edit" and "Create and run". Include, the page will be redirected to the middle page after clicking "Create and edit"; after click "Create and run", the page will not be redirected, but click the VM name could be in the middle page; Check the content of the middle page; Check the button "Go to VMs list" of the middle page.

I am not sure whether it will be easy if we try to move this into an existing test, according to the debug process above. And, if move it into createTest, there will bring more complexity, such as need to consider that os is RHEL. Also, if we didn't figure out why the deletion needs a reload, the failure may spread to all create test